tests/rgw/sanity_rgw.py

```diff
@@ -94,6 +97,22 @@ def run(ceph_cluster, **kw):
             sudo=True, cmd=f"yum install -y {pkgs}", long_running=True
         )
 
+    if install_start_kafka:
+        log.info("install and start kafka broker for bucket notification tests")
+        rgw_node.exec_command(
+            cmd="wget http://magna002.ceph.redhat.com/cephci-jenkins/kafka_2.13-2.8.0.tgz && tar -zxvf kafka_2.13-2.8.0.tgz && mv kafka_2.13-2.8.0 kafka && chown cephuser:cephuser kafka", long_running=True
+        )
+        KAFKA_HOME='/home/cephuser/kafka/'
+        rgw_node.exec_command( check_ec=False, sudo=True,
+        cmd=f"{KAFKA_HOME}/bin/zookeeper-server-start.sh -daemon {KAFKA_HOME}/config/zookeeper.properties")
+
+        #wait for zookeepeer service to start
+        time.sleep(30)
+        rgw_node.exec_command( check_ec=False, sudo=True,
+        cmd=f"{KAFKA_HOME}/bin/kafka-server-start.sh -daemon {KAFKA_HOME}/config/server.properties")
+        # wait for kafka service to start
+        time.sleep(30)
```

Comment:
can you move this into a different function? we can make of this (if needed in the future ) with just an import.

Reply:
@rakeshgm , sure have added the function in utils